CMakeLists.txt

```diff
@@ -432,14 +437,19 @@ if(WIN32)
     list(APPEND MINIZIP_DEF -D_CRT_SECURE_NO_DEPRECATE)
     list(APPEND MINIZIP_SRC mz_os_win32.c mz_strm_os_win32.c)
 
-    if(MZ_PKCRYPT OR MZ_WZAES OR MZ_SIGNING)
+    if(MZ_PKCRYPT OR MZ_WZAES OR MZ_SIGNING OR MZ_BCRYPT)
```

Comment:
I'm not sure we need `MZ_BCRYPT` check here. Crypt is always conditional of `MZ_PKCRYPT OR MZ_WZAES`

Reply:
Removed and checked.